src/main/java/edu/cornell/kfs/fp/businessobject/USBankRecordFieldUtils.java

```diff
@@ -0,0 +1,194 @@
+        String month = sub.substring(4,6);
+        String day = sub.substring(6,8);
+        theDate = Date.valueOf(year+"-"+month+"-"+day);
+    }
+    catch (Exception e) {
+        throw new Exception("Unable to parse date from the value " + line.substring(begin,end) + " on line " + lineCount);
+    }
+    return theDate;
+  }
+
+  /**
+   * Extracts a KualiDecimal from a substring less any ending whitespace for a given beginning and ending position.
+   * 
+   * @param line Superstring
+   * @param begin Beginning index
+   * @param end Ending index
+   * @param lineCount The current line number
+   * @return The KualiDecimal parsed from the trimmed substring
+   * @throws Exception When unable to parse the KualiDecimal
+   */
+  public static KualiDecimal extractDecimal(String line, int begin, int end, int lineCount) throws Exception {
+    KualiDecimal theDecimal;
+    try {
+        String sanitized = line.substring(begin, end);
+        sanitized = StringUtils.remove(sanitized, '-');
+        sanitized = StringUtils.remove(sanitized, '+');
+        theDecimal = new KualiDecimal(sanitized);
+    }
+    catch (Exception e) {
+        throw new Exception("Unable to parse " +  line.substring(begin, end) + " into a decimal value on line " + lineCount);
+    }
+    return theDecimal;
+  }
+
+  /**
+   * Extracts a KualiDecimal representing dollars and cents from a substring less any ending whitespace 
+   * for a given beginning and ending position.
+   * 
+   * @param line Superstring
+   * @param begin Beginning index
+   * @param end Ending index
+   * @param lineCount The current line number
+   * @return The KualiDecimal parsed from the trimmed substring
+   * @throws Exception When unable to parse the KualiDecimal
+   */
+  public static KualiDecimal extractDecimalWithCents(String line, int begin, int end, int lineCount) throws Exception {
+    KualiDecimal theDecimal;
+    KualiDecimal theCents;
+    try {
+        String sanitized = line.substring(begin, end-2);
+        sanitized = StringUtils.remove(sanitized, '-');
+        sanitized = StringUtils.remove(sanitized, '+');
+        theDecimal = new KualiDecimal(sanitized);
+        theCents = new KualiDecimal(line.substring(end-2, end));
+        theCents = theCents.multiply(new KualiDecimal("0.01"));
+        theDecimal = theDecimal.add(theCents);
+    }
+    catch (Exception e) {
+        throw new Exception("Unable to parse " +  line.substring(begin, end) + " into a decimal value on line " + lineCount);
+    }
+    return theDecimal;
+  }
+
+  /**
+   * Extracts a cycle Date from a substring less any ending whitespace for a given beginning and ending position.
+   * 
+   * @param line Superstring
+   * @param begin Beginning index
+   * @param end Ending index
+   * @param lineCount The current line number
+   * @return The cycle Date parsed from the trimmed substring
+   * @throws Exception When unable to parse the cycle Date
+   */
+  @SuppressWarnings("deprecation")
```

Comment:
If you can make the change in less than an hour, I'd say do it. I know we get a lot of warnings now, but I'd rather pay down that tech debt than add to it. org.kuali.kfs.sys.util.KfsDateUtils.newDate() uses Calendar to construct a java.sql.Date in case that's a helpful reference.